js/src/javascript/beautifier.js

```diff
@@ -1057,7 +1057,10 @@ Beautifier.prototype.handle_semicolon = function(current_token) {
 };
 
 Beautifier.prototype.handle_string = function(current_token) {
-  if (this.start_of_statement(current_token)) {
+  var backtickRegex = /^[`].+[`]$/gm;
+  if ((backtickRegex.test(current_token.text) && (this._flags.last_token.type === TOKEN.END_EXPR || this._flags.last_token.type === TOKEN.WORD) && current_token.whitespace_before === '')) {
```

Comment:
Instead of regex-ing for the whole text, would this work? 
```suggestion
  if (current_token.text.startsWith("`") && (this._flags.last_token.type === TOKEN.END_EXPR || this._flags.last_token.type === TOKEN.WORD) && current_token.whitespace_before === '') {
```